Samples/Objective-C/nhubsample-refresh/nhubsample-refresh/NotificationTableViewCell.h

```diff
@@ -0,0 +1,18 @@
+//
+//  NotificationTableViewCell.h
+//  nhubsample-refresh
+//
+//  Created by Artem Egorov on 4/24/20.
```

Comment:
Need to remove the names from the headers and put the MIT license header instead.